### PR TITLE
Update some paths in HACKING

### PR DESCRIPTION
**Pull request by techee**

### CODE REVIEW

File: HACKING [hunk 1]
```diff
@@ -658,7 +658,7 @@ https://github.com/universal-ctags/ctags
 
 Method
 ``````
-* Add foo.c to SRCS in Makefile.am and meson.build.
+* Add foo.c to SRCS in ctags/Makefile.am and meson.build.
```
Comment (b4n) [inline HACKING:661]: It's in `parsers` inside `ctags/Makefile.am`
```suggestion
* Add ctags/parsers/foo.c to ``parsers`` in ctags/Makefile.am and to the
  ``ctags`` library in meson.build.
```

**Comment by elextr** [comment]
LGBI and @b4n's suggestion.

File: HACKING (hunk 1 above)
Comment (techee) [reply HACKING]: Done, thanks.

**Review by b4n — APPROVED** [review]
LGBI

**Pull request merged**